src/app/features/users-page/users-page.component.scss

```diff
@@ -9,6 +9,9 @@
     }
 }
 
-.table-col {
-    width: 150px;
+.open-filter {
+    cursor: pointer;
+    width: 1000px;
```

Comment:
why is your button so big?
make is default size

if you wanted to make it 100% - set 100%, big static sizes aren't responsive!!!